smalltalksrc/VMMaker/VMStackPage.class.st

```diff
@@ -34,6 +34,15 @@ VMStackPage class >> alignedByteSizeOf: anObject forClient: aVMClass [
 	^self surrogateClass alignedByteSize
 ]
 
+{ #category : #translation }
+VMStackPage class >> filteredInstVarNames [
+	| instVarNames |
+	
+	instVarNames := super filteredInstVarNames.
+	
+	^BytesPerWord = 4 ifTrue:  [ ^instVarNames copyWithout: #padToWord ] ifFalse: [ ^instVarNames ]
+]
```

Comment:
used this method for "ignoring" the `padToWord` but it probable broke something since now although I can traspile I cant run the generated code

Reply:
fixed, the issue was cause by a enumerating the type def dictionary instead of the instance variables